src/ocf_core_priv.h

```diff
@@ -58,4 +111,7 @@ int ocf_core_volume_type_init(ocf_ctx_t ctx);
 
 void ocf_core_volume_type_deinit(ocf_ctx_t ctx);
 
+#define for_each_core(_cache, _core, _id) \
+	for (_id = 0; _core = &cache->core[_id], _id < OCF_CORE_MAX; _id++)
```

Comment:
Why not iterate over added cores only, like it used to be done?

Reply:
This was meant to be as much generic as possible. We may add specialized version e.g. `for_each_core_added` if we want to.

Reply:
After all I decided to special case iterating over all cores, because it's used much less often and making it default would be error-prone.